pom.xml

```diff
@@ -30,7 +30,7 @@
 
     <groupId>io.gravitee.policy</groupId>
     <artifactId>gravitee-policy-transform-status-code</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.0-alpha.1</version>
```

Comment:
you can revert this change, the version will be  handled automatically

Reply:
Done